scripts/rename

```diff
@@ -14,21 +14,26 @@ fi
 for file in $(find . -not \( -name .git -prune \) -a -type f); do
     file_name=$(basename "$file")
     case $file_name in
-        CONTRIBUTING.MD|README.md|rename|setup.cfg|publish|installation_and_basic_usage.ipynb)
+        rename)
             echo "Skipping $file"
             continue;;
     esac
     set +e
-    sed ${sedflags} -e "s/stactools-package/stactools-${name}/g" $file
-    sed ${sedflags} -e "s/stactools\.package/stactools.${underscore_name}/g" $file
-    sed ${sedflags} -e "s/stactools_package/stactools_${underscore_name}/g" $file
-    sed ${sedflags} -e "s;stactools/package;stactools/${underscore_name};g" $file
-    sed ${sedflags} -e "s/package-name/${name}/g" $file
+    sed ${sedflags} -e "s/stactools-ephemeral/stactools-${name}/g" $file
+    sed ${sedflags} -e "s/stactools\.ephemeral/stactools.${underscore_name}/g" $file
+    sed ${sedflags} -e "s/stactools_ephemeral/stactools_${underscore_name}/g" $file
+    sed ${sedflags} -e "s;stactools/ephemeral;stactools/${underscore_name};g" $file
+    sed ${sedflags} -e "s/ephemeral/${name}/g" $file
     set -e
 done
 
-if [ -d src/stactools/package ]; then
-    git mv src/stactools/package "src/stactools/$underscore_name"
+if [ -f README-template.md ]; then
+    git rm README.md
+    git mv README-template.md README.md
```

Comment:
If people are following along with the template's README, it might be confusing if one of the instructions deletes the file they're following along with? I might leave this a manual process.

Reply:
Yeah, I was assuming the instructions would would be in the template. Since we're dropping that, I'll remove this.